prod.Dockerfile

```diff
@@ -17,23 +17,33 @@ RUN set -o pipefail && \
 
 ENV PROJECT_HOME /usr/src
 ENV PROJECT_NAME universal-application-tool-0.0.1
+ENV PROJECT_LOC "${PROJECT_HOME}/${PROJECT_NAME}"
 
-COPY "${PROJECT_NAME}" "${PROJECT_HOME}/${PROJECT_NAME}"
-RUN cd "${PROJECT_HOME}/${PROJECT_NAME}" && \
+COPY "${PROJECT_NAME}"
+RUN cd "${PROJECT_LOC}" && \
     npm install -g npm@8.5.1 && \
     npm install && \
     sbt update && \
-    sbt dist
+    sbt dist && \
+    mv "${PROJECT_LOC}/target/universal/universal-application-tool-0.0.1.zip" /civiform.zip && \
+    unzip /civiform.zip &&
+    chmod +x /universal-application-tool-0.0.1/bin/universal-application-tool
 
-# This is a common trick to shrink container sizes.  we just throw away all that build stuff and use only the jars
-# we built with sbt dist.
+# This is a common trick to shrink container sizes. We discard everything added
+# during the build phase and use only the inflated artifacts created by sbt dist.
 FROM adoptopenjdk/openjdk11:jdk-11.0.10_9-alpine-slim AS stage2
-COPY --from=stage1 /usr/src/universal-application-tool-0.0.1/target/universal/universal-application-tool-0.0.1.zip /civiform.zip
+COPY --from=stage1 /universal-application-tool-0.0.1 /universal-application-tool-0.0.1
+
+# Upgrade packages for stage2 to include latest versions.
+RUN set -o pipefail && \
+    apk update && \
+    apk add --upgrade apk-tools && \
+    apk upgrade --available && \
+    apk add --no-cache --update bash openssh
```

Comment:
Is this updating the image OS or our application?

Reply:
It's updating the image